docs/docs/installation/rasa-pro/installation.mdx

```diff
@@ -96,36 +83,36 @@ For virtual environments:
 - Linux and macOS: `$VIRTUAL_ENV/pip.conf`
 - Windows: `%VIRTUAL_ENV%\pip.ini`
 
-
 ```
 [global]
 extra-index-url = https://europe-west3-python.pkg.dev/rasa-releases/rasa-plus-py/simple/
-
 ```
 
 Finally, you should be able to run `pip install rasa-plus`.
 
 #### Installing with `poetry`
 
-To install `rasa-plus` with `poetry`, you will need to associate the Artifact Registry URL with `rasa-plus` before installing it.
-Note that you must upgrade poetry to the latest minor (`1.2.0`) in order for `poetry` to work with the GCP authentication set-up.
-Proceed with the following steps:
+:::note
+The following configuration applies to the current stable version of `poetry`,
+which is `1.6.1` at this time.
+:::
+
+To install `rasa-plus` with `poetry`, you will need to associate the Artifact
+Registry URL with `rasa-plus` before installing it in order to authenticate
+with GCP. Proceed with the following steps:
 
-1. Run `poetry self add "keyrings.google-artifactregistry-auth"`
+1. Run `poetry self add keyrings.google-artifactregistry-auth`
 2. Add this section to `pyproject.toml`:
 
 ```toml
 [[tool.poetry.source]]
 name = "rasa-plus"
 url = "https://europe-west3-python.pkg.dev/rasa-releases/rasa-plus-py/simple"
-default = false
-secondary = true
-
+priority = "supplemental"
```

Comment:
This works with 1.5 too, [as per their docs](https://python-poetry.org/docs/repositories#supplemental-package-sources), so I think we should mention it.

The issue though is that internally at Rasa we are still using poetry 1.4. I'd suggest to have a tab with the different toml instructions based on the poetry version. Thoughts?